tests/operator/operator.go

```diff
@@ -3275,3 +3212,65 @@ spec:
 	}
 	return vmYamls
 }
+
+func generatePreviousVersionVmsnapshotYamls(virtClient kubecli.KubevirtClient, vmYamls map[string]*vmYamlDefinition, workDir string) {
```

Comment:
nit: I think the function name doesnt express what this function actually does, it says generate but return nothing, it actually update the given VM-YAMLs with some "data".
How about calling it something like `populateVmYamlSnapshot`/`updateVmYamlSnapshot`?

Reply:
No, it does generates. it creates actual files.